config/pepperConfig.js.ctmpl

```diff
@@ -259,3 +259,19 @@ var DDP_ENV = {
         {{end}}
     {{end}}
 {{end}}
+
+{{if eq $study_guid "circadia"}}
+    {{if eq $environment "local"}}
+        DDP_ENV['auth0ClientId'] = "GMMFtLjB1c38Fmr3UVEm8WqBsvCIeJ9d";
+        DDP_ENV['auth0Domain'] = "circadia-dev.us.auth0.com";
+        DDP_ENV['auth0Audience'] = "circadia-dev.us.auth0.com";
+        DDP_ENV['basePepperUrl'] = "http://localhost:5556";
+    {{end}}
+
+    {{with secret (printf "secret/pepper/%s/%s/circadia/conf" $environment $version)}}
+        DDP_ENV['auth0ClientId'] = "{{.Data.auth0.clientId}}";
+        DDP_ENV['auth0Domain'] = "{{.Data.auth0.loginDomain}}";
+        DDP_ENV['auth0Audience'] = "{{.Data.auth0.loginDomain}}";
+        DDP_ENV['basePepperUrl'] = "{{.Data.basePepperUrl}}";
```

Comment:
I think we'll be calling it `serverBaseUrl` in Vault.
```suggestion
        DDP_ENV['basePepperUrl'] = "{{.Data. serverBaseUrl}}";
```

Reply:
Done :)